specification/attributes/column_naming_convention.md

```diff
@@ -19,16 +19,18 @@ Naming convention for columns appearing in billing data.
 
 ## Requirements
 
-* Column IDs MUST use [Pascal case](https://techterms.com/definition/pascalcase).
-* Column IDs MUST NOT use abbreviations.
-* Column IDs SHOULD NOT use acronyms.
-* Column IDs MUST be alphanumeric with no special characters.
-* Columns that have an ID and a Name MUST have the Id or Name suffix in the Column ID. Display Name for a Column MAY
-  avoid the Name suffix if it is considered superfluous
+- All columns defined by FOCUS MUST follow the following rules:
+  - Column IDs MUST use [Pascal case](https://techterms.com/definition/pascalcase).
+  - Column IDs MUST NOT use abbreviations.
```

Comment:
```suggestion
  - Column IDs MUST NOT use abbreviations beyond exceptions listed within the Column Naming Convention guidelines. 
```

Reply:
Does this mean we need to add this caveat for every exception? I was assuming the exception was enough to call out that it's overriding the defined rules 🤔 Ultimately, I'm fine either way.

Reply:
Given this is separate from the prefix, I'm going to "won't fix" this one. Let's chat in Slack. If you feel strongly, I'm happy to push out a separate PR.